package.json

```diff
@@ -16,28 +16,24 @@
   },
   "author": "Josh Sommer <joshdsommer@gmail.com> (https://twitter.com/_JoshSommer)",
   "license": "ISC",
-  "dependencies": {},
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/cli": "6.x",
-    "@angular/common": "6.x",
-    "@angular/compiler": "6.x",
-    "@angular/compiler-cli": "6.x",
-    "@angular/core": "6.x",
-    "@angular/forms": "6.x",
-    "@angular/http": "6.x",
-    "@angular/platform-browser": "6.x",
-    "@angular/platform-browser-dynamic": "6.x",
+    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular/cli": "7.3.10",
+    "@angular/common": "7.2.16",
+    "@angular/compiler": "7.2.16",
+    "@angular/compiler-cli": "7.2.16",
+    "@angular/core": "7.2.16",
+    "@angular/forms": "7.2.16",
+    "@angular/http": "7.2.16",
+    "@angular/platform-browser": "7.2.16",
+    "@angular/platform-browser-dynamic": "7.2.16",
     "ng-packagr": "^5.7.1",
-    "rxjs": "6.5.3",
-    "typescript": "2.9.2",
-    "zone.js": "~0.8.26"
-  },
-  "peerDependencies": {
-    "@angular/common": ">=2.x",
-    "@angular/core": ">=2.x",
-    "@angular/forms": ">=2.x"
+    "rxjs": "6.6.7",
+    "typescript": "3.2.4",
+    "zone.js": "~0.8.26",
+    "rxjs-tslint": "^0.1.8"
   },
+  "peerDependencies": {},
```

Comment:
Hmm, I think the `peerDependencies` are important so that consuming projects know they have to install certain versions of specific Angular projects.

Reply:
[dependencies](https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#dependencies) are installed on both:

npm install from a directory that contains package.json
npm install $package on any other directory

[devDependencies](https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#devdependencies) are:

also installed on npm install on a directory that contains package.json, unless you pass the --production flag (go upvote [Gayan Charith's answer](https://stackoverflow.com/a/31229205/895245)).
not installed on npm install "$package" on any other directory, unless you give it the --dev option.
are not installed transitively.

[peerDependencies](https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#peerdependencies):

Give a warning if missing on npm install, and you have to solve the dependency yourself manually. When running, if the dependency is missing, you get an error 
peerDependencies are automatically installed unless an upstream dependency conflict is present that cannot be automatically resolved

Reply:
Those are the general definitions.  Regardless, I did not do anything to manually/deliberately remove the peerDependencies, and everything seems to function properly after "npm i" or "npm install".

Reply:
Yep, understood.  There are peer dependencies in the version wherein I rolled up all the angular updates.  There are none here and I don't have a good explanation for that except that perhaps they are added as updates/upgrades happen.  

Yes, each alteration/step was accompanied by a "run demo" command to make sure everything was still working properly.  It all functions as desired.

Reply:
From the Angular docs:
https://github.com/RenovoSolutions/ngx-datetimepicker/pull/146/commits/4f43868eaff032d38100875e8dfae04648a7fbed
> The Angular version used to build an application should always be the same or greater than the Angular versions used to build any of its dependent libraries.

So, it feels like this number's going to need to be bumped to `7.2.x`.

Granted, this passage of the docs seems to be more relevant to Angular v12/v13. But I think it's still probably accurate advice.

Reply:
Updated these appropriately